rules/allied-vehicles.yaml

```diff
@@ -61,8 +61,7 @@ cmin:
 		HarvestVoice: Move
 		DeliverVoice: Move
 	ChronoResourceDelivery:
-		WarpInSequence: # unused
```

Comment:
You're wrong, the warpin.shp was unused and the game didn't played a warpanim in the refinery. The example gif in https://github.com/OpenRA/ra2/issues/134 showcases this quite well.